src/Microsoft.VisualStudio.Services.Agent/TraceManager.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Concurrent;
 using System.Diagnostics;
 using Agent.Sdk.SecretMasking;
+using ISecretMaskerVSO = Microsoft.TeamFoundation.DistributedTask.Logging.ISecretMasker;
```

Comment:
I don't know if it would be better or not, but most files don't need to use VSO suffixed alias since most files don't need to interact with Microsoft.Security.Utilities.Core.

Reply:
Let's do it, less code change is good.